src/System.Binary.Base64/System/Binary/Base64Decoder.cs

```diff
@@ -307,21 +225,21 @@ public static int DecodeInPlace(Span<byte> buffer)
 
             for (; si < buffer.Length - (padding != 0 ? 4 : 0);)
             {
-                Decode(buffer[si++], buffer[si++], buffer[si++], buffer[si++], out r0, out r1, out r2);
+                /*Decode(buffer[si++], buffer[si++], buffer[si++], buffer[si++], out r0, out r1, out r2);
```

Comment:
The Decode API that takes individual bytes is gone. I will re-write this method in future PRs which works with the new decoding map.